client-v2/src/shared/actions/ArticleFragments.ts

```diff
@@ -177,20 +177,20 @@ const getArticleEntitiesFromDrop = async (
   const isURL = isValidURL(resourceIdOrUrl);
   const id = isURL ? getIdFromURL(resourceIdOrUrl) : resourceIdOrUrl;
   const isNonGuLink = isURL && !id;
-  if (isNonGuLink) {
+  const meta = getArticleFragmentMetaFromUrlParams(resourceIdOrUrl);
+  if (isNonGuLink && !meta) {
```

Comment:
Rename `isNonGuLink` to be more specific

Reply:
👍